src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,79 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.PrePersist;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "ticket_id")
+    private List<Ticket> tickets;
```

Comment:
```suggestion
    @JoinColumn(name = "order_id")
    private List<Ticket> tickets;
```
because each ticket db row will have 1 `order_id` column to store the relation to 1 order(many tickets to one order)
By that, the order will have multiple tickets(one order to many tickets)